test/integration/progressUsers.test.js

```diff
@@ -0,0 +1,187 @@
+const chai = require("chai");
+const sinon = require("sinon");
+
+const firestore = require("../../utils/firestore");
+const app = require("../../server");
+const authService = require("../../services/authService");
+
+const addUser = require("../utils/addUser");
+const cleanDb = require("../utils/cleanDb");
+const {
+  standupProgressDay1,
+  standupProgressDay2,
+  incompleteProgress,
+  stubbedModelProgressData,
+} = require("../fixtures/progress/userProgress");
+
+const userData = require("../fixtures/user/user")();
+
+const cookieName = config.get("userToken.cookieName");
+const { expect } = chai;
+
+// eslint-disable-next-line mocha/no-skipped-tests
+describe.skip("Test Progress Updates API", function () {
+  afterEach(async function () {
+    await cleanDb();
+  });
+
+  describe("Test the POST Call", function () {
+    let clock;
+    let testUserId;
+    let testUserJwt;
+    beforeEach(async function () {
+      clock = sinon.useFakeTimers({
+        now: new Date(2023, 5, 2).getTime(),
+        toFake: ["Date"],
+      });
+      testUserId = await addUser(userData[1]);
+      testUserJwt = authService.generateAuthToken({ userId: testUserId });
+    });
+
+    afterEach(function () {
+      clock.restore();
+    });
+
+    it("Stores the initial progress entry for the day and disregards any subsequent entries made before 6 am of the next day", async function () {
+      const responseFirstDay = await chai
+        .request(app)
+        .post(`progresses`)
+        .set("Cookie", `${cookieName}=${testUserJwt}`)
+        .send(standupProgressDay1);
+
+      expect(responseFirstDay).to.have.status(200);
+      expect(responseFirstDay.body).to.have.keys(["message", "data"]);
+      expect(responseFirstDay.body.data).to.have.keys([
+        "id",
+        "userId",
+        "type",
+        "completed",
+        "planned",
+        "blockers",
```

Comment:
do these keys sound good for user progress? it looks good for task progress. Completed key can change to `progress`